commit e321435ac993e5e0a280263a1c702cec2b671e7c
Author: Sergio Martins <iamsergio@gmail.com>
Date:   Sat May 2 23:15:00 2015 +0100

    Don't allocate a char[] with an undefined size.
    
    packetSize is still undefined at this point.

diff --git a/phonon/experimental/packetpool.cpp b/phonon/experimental/packetpool.cpp
index a1effe1..8ebe134 100644
--- a/phonon/experimental/packetpool.cpp
+++ b/phonon/experimental/packetpool.cpp
@@ -35,7 +35,7 @@ int PacketPool::unusedPackets() const { return d_ptr->ringBufferSize; }
 
 PacketPoolPrivate::PacketPoolPrivate(int _packetSize, int _poolSize)
     : freePackets(new PacketPrivate *[_poolSize]),
-    packetMemory(new char[packetSize * _poolSize]),
+    packetMemory(new char[_packetSize * _poolSize]),
     readPosition(0), writePosition(0),
     ringBufferSize(_poolSize),
     packetSize(_packetSize),
